packages/sdks/src/functions/get-content/index.ts

```diff
@@ -47,6 +47,13 @@ type ContentResponse =
     };
 
 const _fetchContent = async (options: GetContentOptions) => {
+  // Set noTraverse=true if NOT already passed by user, for query performance
+  if (options.limit === undefined || options.limit > 1) {
```

Comment:
so if I understand correctly, when `limit === =`, then we don't want to set `noTraverse = true`? I'm surprised this condition is in gen2 but not gen1. Any reason for that?

Reply:
in Gen1, there is a separate `getAll` call, which is where the change is made. In Gen2, from my understanding, there is a common `getContent` call used for both individual and multiple fetches.

Reply:
I see. Can we move this logic to `generateContentUrl`? This is where we set the default values for all of our `options`. It is a more ideal place to have this logic in

Reply:
I understand the benefit, but I cant move it coz `generateContentUrl` assumes a default limit of 30 which affect my condition to apply `noTraverse`. It should only apply in case of `fetchEntries` and not to the single `fetchEntry` case.

Reply:
nvmd, found a way where it was possible. Moved it. pls have a look

Reply:
@samijaber @shyam-builder

Reply:
Awesome, thanks @murtaza-builder! Your build is failing because you need to update the snapshot tests for generateContentUrl